packages/forms/src/templates/ArrayFieldTemplate.scss

```diff
@@ -87,23 +108,25 @@ $tf-filterwidget-shadow-color: $shadow !default;
 			}
 
 			fieldset {
-				> div:first-child {
-					border-bottom: 1px solid $wild-sand;
+				> :global(.form-group:nth-child(2)) {
```

Comment:
I'm not sure this work well on all supported browser

Reply:
https://caniuse.com/#feat=css-sel3